Author: Andreas Beckmann <anbe@debian.org>
Description: do not fail to find scenes if SCENE_DIR is a symlink

--- a/unix/scripts/allanim.sh
+++ b/unix/scripts/allanim.sh
@@ -111,7 +111,7 @@ fi
 
 SCRIPT_DIR=`dirname "$0"`
 
-FILE_LIST=`find "$SCENE_DIR" -path "*animations*" -name "*.pov" | sort`
+FILE_LIST=`find "$SCENE_DIR/" -path "*animations*" -name "*.pov" | sort`
 
 if [ -z "$LOG_FILE" ] ; then
   echo "$FILE_LIST" | xargs -n 1 "$SCRIPT_DIR/render_anim.sh" "$OUTPUT_DIR"
--- a/unix/scripts/allscene.sh
+++ b/unix/scripts/allscene.sh
@@ -117,9 +117,9 @@ fi
 SCRIPT_DIR=`dirname "$0"`
 
 if [ -z "$RENDER_ALL" ] ; then
-  FILE_LIST=`find "$SCENE_DIR" -not -path "*animations*" -not -path "*portfolio*" -name "*.pov" | sort -r |  xargs grep -l -E '^//[ ]+[-+]{1}[^ -]'`
+  FILE_LIST=`find "$SCENE_DIR/" -not -path "*animations*" -not -path "*portfolio*" -name "*.pov" | sort -r |  xargs grep -l -E '^//[ ]+[-+]{1}[^ -]'`
 else
-  FILE_LIST=`find "$SCENE_DIR" -not -path "*animations*" -not -path "*portfolio*" -name "*.pov" | sort -r`
+  FILE_LIST=`find "$SCENE_DIR/" -not -path "*animations*" -not -path "*portfolio*" -name "*.pov" | sort -r`
 fi
 
 if [ -z "$LOG_FILE" ] ; then
--- a/unix/scripts/portfolio.sh
+++ b/unix/scripts/portfolio.sh
@@ -120,7 +120,7 @@ CURR_DIR=`pwd`
 SCENE_DIR=`echo "$SCENE_DIR" | sed "s?^\.?$CURR_DIR?"`
 SCENE_DIR=`echo "$SCENE_DIR" | sed "s?^\([^/]\)?$CURR_DIR/\1?"`
 
-FILE_LIST=`find "$SCENE_DIR" -not -path "*__empty*" -name "*.ini" | sort`
+FILE_LIST=`find "$SCENE_DIR/" -not -path "*__empty*" -name "*.ini" | sort`
 
 cd "$OUTPUT_DIR"
 
